deeppavlov/models/go_bot/nlg/mock_json_nlg_manager.py

```diff
@@ -197,10 +197,14 @@ def decode_response(self,
         response = JSONNLGResponse(slots_values, actions_tuple)
         verbose_response = VerboseJSONNLGResponse.from_json_nlg_response(response)
         verbose_response.policy_prediction = policy_prediction
-        verbose_response._nlu_responses = utterance_batch_features._nlu_responses
         response_text = self.generate_template(verbose_response)
         verbose_response.text = response_text
-        return verbose_response
+        if utterance_batch_features:
+            verbose_response._nlu_responses = utterance_batch_features._nlu_responses
+            return verbose_response
+        # TripPy Case - Use same return type as nlg_manager, i.e. str
+        else:
+            return verbose_response.text
```

Comment:
still I believe that this change should be reversed: all the model specific logic should be done in the model code

Reply:
Alternatively what about we wrap the normal nlg manage as a verbose_response object? So that the text of both can be accessed via .text
I just think the return type for decode() should be the same if its the json nlg manager or the normal nlg manager

Reply:
lets discuss it in zoom